lib/internal/test_runner/runner.js

```diff
@@ -503,8 +502,12 @@ function run(options = kEmptyObject) {
   let filesWatcher;
   const opts = { __proto__: null, root, signal, inspectPort, testNamePatterns, only };
   if (watch) {
+    root.harness.watchMode = true;
     filesWatcher = watchFiles(testFiles, opts);
-    postRun = undefined;
+    postRun = () => {
+      kFilterArgs.delete('--experimental-test-coverage');
```

Comment:
Why do you remove this flag here?

Reply:
This will help to re-run the watching file with code coverage enabled ([filterExecArgv()](https://github.com/nodejs/node/pull/51288/files#diff-840cee72cdda14be90e6c5607ffa844eaf73be966607ddc834cc667bd142fb21R110-R113)). without this the watched file re-runs without code coverage.

Reply:
for the initial run of all test files, the coverage is accumelated and printed by the orcastator process.
after the initial run ends - for every watched file, we pass `--experimental-test-coverage` so that the file calculates its individual coverage internally